### ISSUE TITLE
Missing emergency airlocks

### ISSUE THREAD

**Issue by LifesDisciple** [Aurorastation/Aurora.3#4444]
Emergency airlocks missing. THe main door to the forensics office and windows between the office and the lab

**Comment by Arrow768** [comment]
Do you mean the yellow emergency shutters ?

**Comment by LifesDisciple** [comment]
Yes, the thingies that come down in case of depressurizing or when a fire alarm is set off
